### PR TITLE
Relax home screen

### LINKED ISSUE #447: Relax home screen
**Actual Behaviour.**  
Missing relax home screen.

**Expected Behaviour**  
Develop a nice home screen similar to Focus mode.

**Would you like to work on the issue?**
Issue for Codeheat.

### PR DESCRIPTION
**Pull request by kshitiz-kumar**
Fixes #447

**Changes**: Added Relax home screen

**Screenshot/s for the changes**: <br>
> Relax Home Screen
 
<image src = "https://user-images.githubusercontent.com/38812752/64953866-a456d880-d8a1-11e9-92ec-6c63e7922aa3.png"  width = "200"  height ="350">

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: [app-debug.zip](https://github.com/fossasia/neurolab-android/files/3638359/app-debug.zip)

### CODE REVIEW

File: app/src/main/java/io/neurolab/activities/ProgramModeActivity.java [hunk 1]
```diff
@@ -95,6 +95,8 @@ public void onBackPressed() {
         super.onBackPressed();
         if (mode == FOCUS_PROGRAM_MODE)
             startActivity(new Intent(this, FocusParentActivity.class));
+        if(mode == RELAX_PROGRAM_MODE)
+            startActivity(new Intent(this,RelaxParentActivity.class));
```
Comment (jddeep) [inline app/src/main/java/io/neurolab/activities/ProgramModeActivity.java]: Please reformat your code at all places. Use Android studio features of rearrange and reformat for this.

**Review by jddeep — CHANGES_REQUESTED** [review]

**Comment by jddeep** [comment]
Also please provide an updated apk for testing.

File: app/src/main/res/layout/activity_relax_parent.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="150dp"
+        android:background="@drawable/gradient_relax">
+
+        <ImageView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:scaleType="fitXY"
```
Comment (adityastic) [inline app/src/main/res/layout/activity_relax_parent.xml]: centerCrop plz

**Review by adityastic — CHANGES_REQUESTED** [review]
Provide apk for test

**Comment by adityastic** [comment]
Please also follow Semantic naming when creating a PR, Reference:
https://github.com/probot/semantic-pull-requests

**Comment by kshitiz-kumar** [comment]
I have provided the apk for testing also made the changes requested. Also I am new to this open source culture, therefore I really don' t know about the best practices. Any suggestion for that is really appreciated.

File: app/src/main/res/layout/activity_relax_parent.xml (hunk 2 above)
Comment (kshitiz-kumar) [reply app/src/main/res/layout/activity_relax_parent.xml]: @adityastic done

**Comment by jddeep** [comment]
@kshitiz-kumar The apk is not getting installed. Please upload a correct working apk. Zip the apk file and upload it in the PR description body directly. No need to upload to Google drive.

**Comment by kshitiz-kumar** [comment]
@jddeep please check the updated apk

**Comment by jddeep** [comment]
@kshitiz-kumar The apk is still not getting installed. Please check yourself by trying to install the apk you provided.

**Comment by kshitiz-kumar** [comment]
@jddeep I have installed on three different phone and it is working

**Review by jddeep — APPROVED** [review]
Good job :+1:

**Pull request merged**